libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/camera/CameraComponent.kt

```diff
@@ -219,7 +220,8 @@ internal class CameraComponent constructor(
                 viewportDataSource.evaluate()
             }
         }
-        coroutineScope.launch {
+        // using immediate dispatcher causes nested store updates
```

Comment:
That's a temporary solution, right? Could you add a TODO with a ticket?

Reply:
could also consider this the follow up https://github.com/mapbox/mapbox-navigation-android/pull/6442

Reply:
Yeah, I hope we come up with something different. But not everyone agrees here.
So no TODO for now. There's another PR open already, so we won't forget about this.